vue-frontend/finalize.js

```diff
@@ -47,3 +48,18 @@ const uint8_t DASH_HTML[] PROGMEM = { ${indexHTML} };
 
 
 fs.writeFileSync(__dirname+'/dist/webpage.h', source, 'utf8');
+
+// Produce a second variant with zopfli
+// Zopfli is a improved zip algorithm by google
+// Takes much more time and maybe is not available on every machine
+const input =  html;
+gzip(input, {numiterations: 15}, (err, output) => {
+    indexHTML = output;
```

Comment:
Convert this to Byte Array using `getByteArray()` and did you test this gzip compression with an iphone? Safari creates a havoc with different gzip algos, most of the time it will just reject to load the page. Let me know if it works...

Reply:
The output is byte array. Zopfli is compatible to default gzip.
It works with IE11 and FF so should work with safari.

Reply:
Okay. I will merge it.